pages/signup/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+    FormControl,
+    Input,
+    InputLabel,
+    FormHelperText,
+    Container,
+    Button
+} from "@mui/material"
+
+const signup = () => {
+    return (
+        <Container id="signUpForm" className="signUpForm" maxWidth="md">
+            <FormControl variant="outlined" margin="normal" fullWidth>
+                <InputLabel htmlFor="id">아이디</InputLabel>
+                <Input id="id" aria-describedby="idHelperText" />
+                <FormHelperText id="idHelperText"></FormHelperText>
+            </FormControl>
+
+            <FormControl variant="outlined" margin="normal" fullWidth>
+                <InputLabel htmlFor="password">비밀번호</InputLabel>
+                <Input id="password" aria-describedby="passwordHelperText" />
```

Comment:
```suggestion
                <Input id="password" aria-describedby="passwordHelperText" type="password"/>
```
타입을 password로 선언하면 ****로 표시 됩니다.

<img width="166" alt="스크린샷 2022-01-25 오후 10 51 09" src="https://user-images.githubusercontent.com/37958836/150989300-0cc1abac-ac27-43ef-9d7c-4a896e6943eb.png">

Reply:
넵 수정하겠습니당